CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ include(NuToMacros)
 set_compiler_flags()
 get_git_head_revision(git_refspec commit_nr)
 check_for_dependencies()
+#setup_google_benchmark()
```

Comment:
Lets just remove this.

Reply:
Ups, yeah, forgot it. As you can see its already disabled by a comment :stuck_out_tongue_closed_eyes: